src/gui/editors/AutomationEditor.cpp

```diff
@@ -1276,14 +1005,10 @@ void AutomationEditor::paintEvent(QPaintEvent * pe )
 			for( int level = (int)m_bottomLevel; level <= m_topLevel; level++)
 			{
 				y =  yCoordOfLevel( (float)level );
-				if( level % 10 == 0 )
-				{
-					p.setPen( beatLineColor() );
-				}
-				else
-				{
-					p.setPen( lineColor() );
-				}
+
+				if (level % 10 == 0) { p.setPen(m_beatLineColor); }
+				else { p.setPen(m_lineColor); }
```

Comment:
Just FYI, you can do like this.
```suggestion
				p.setPen(level % 10 == 0 ? m_beatLineColor : m_lineColor);
```